### ISSUE TITLE
wp-env: add ability to follow only PHP error logs

### ISSUE THREAD

**Issue by creativecoder** [WordPress/gutenberg#32565]
## What problem does this address?

Currently when using wp-env, the `logs` command aggregates the logs from all services (because this is what [docker-compose logs](https://docs.docker.com/compose/reference/logs/) does).

When debugging PHP, it can be hard to parse the PHP specific logs from everything else, particularly the http traffic.

## What is your proposed solution?

Add a command to wp-env to follow only the php error logs. It could be an option added to the existing `logs` command or a separate `php-errors` command.

---

I'm not sure what the options are for better configuring the docker environment to support this. Another solution might be enabling `WP_DEBUG_LOG` and following the `wp-content/debug.log` file.

**Comment by westonruter** [comment]
> When debugging PHP, it can be hard to parse the PHP specific logs from everything else, particularly the http traffic.

Additionally, even the PHP-specific logs have a lot of extra noise. When just tailing a regular PHP error log (`tail -f wp-content/debug.log`), I get:

```
[07-Nov-2023 17:44:29 UTC] PHP Notice:  Hello world in /var/www/html/wp-content/plugins/performance/modules/images/image-loading-optimization/storage.php on line 88
```

But when I watch the wp-env logs (`npm run wp-env logs development -- --watch`), I get a lot of other noise (e.g. container name, process ID, client IP, and referrer):

```
wordpress_1        | [Tue Nov 07 17:48:25.932339 2023] [php:notice] [pid 19] [client 172.22.0.1:54766] PHP Notice:  Hello world in /var/www/html/wp-content/plugins/performance/modules/images/image-loading-optimization/storage.php on line 88, referer: http://localhost:8888/?p=33
```

But as @creativecoder notes, the log includes all non-PHP log entries as well. To filter down to just PHP-specific logs I need to do:

```bash
npm run wp-env logs development -- --watch | grep '\[php:'
```

But the other noise in the logging remains.

What I've just started doing is modifying my `.wp-env.json` (or `.wp-env.override.json`) to add a `"WP_DEBUG_LOG":true` entry to the [`config` object](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-env/#wp-env-json). (And then restart Docker via `npm run wp-env start`.) I also found that I needed to `touch` the log file and make it writable or at least make the `wp-content` directory writable, for example:

```bash
chmod 777 $(npm --silent run wp-env install-path 2> /dev/null)/WordPress/wp-content
```

With that in place, I'm now able to tail the error log:

```bash
tail -f $(npm --silent run wp-env install-path 2> /dev/null)/WordPress/wp-content/debug.log
```

I'm using the Linux environment in a Chromebook.
